bgpd/bgp_zebra.c

```diff
@@ -3208,10 +3208,10 @@ static int bgp_zebra_process_srv6_locator_delete(ZAPI_CALLBACK_ARGS)
 	struct srv6_locator loc = {};
 	struct bgp *bgp = bgp_get_default();
 	struct listnode *node, *nnode;
-	struct srv6_locator_chunk *chunk;
+	struct srv6_locator_chunk *chunk, **tovpn_sid_locator_ptr;
```

Comment:
Double pointer is really needed here?

Reply:
Actually, it's not needed and I just introduced it to avoid writing the same long code. I fixed it.